sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/CaseService.java

```diff
@@ -72,6 +73,36 @@ public Case buildGeneratedCase() {
         .build();
   }
 
+  public Case buildGeneratedCaseDE() {
+    firstName = faker.name().firstName();
+    lastName = faker.name().lastName();
+
+    return Case.builder()
+        .firstName(firstName)
+        .lastName(lastName)
+        .caseOrigin("IM LAND")
+        .dateOfReport(LocalDate.now().minusDays(1))
+        .externalId(UUID.randomUUID().toString())
+        .disease("COVID-19")
```

Comment:
random disease

Reply:
I have no translated diseases from EN to DE. I don't wanna make mess, Alwin said to skip it. I don't change it right now